crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -1259,6 +1267,10 @@ pub struct ChangeTrackersState<T> {
     marker: PhantomData<T>,
 }
 
+// SAFETY: WriteState only contains a ComponentId, should be safe to read
+// from multiple threads concurrently
+unsafe impl<T: Component> Sync for ChangeTrackersState<T> {}
```

Comment:
Why not change the `PhantomData` fields instead of using `unsafe` impls?

I.e. `marker: PhantomData<fn()->T>`, which should be `Sync`

Reply:
Done.